.eslintrc.js

```diff
@@ -36,6 +36,11 @@ module.exports = {
       "indent": ["error", 2, {"VariableDeclarator":2}],
       // seems like a good idea not to use explicit undefined
       "no-undefined": "error",
+      // ensure import specifier contains file extension
+      "import/extensions": ["error", "always"],
+      // make sure there is no Node.js specific API slipping into the source files
+      "import/no-nodejs-modules": "error",
+      "import/no-commonjs": "error",
```

Comment:
Do we need to add a npm library for these?  Where can I find the docs?  I don't see these on the main ESLIst list of rules?

Reply:
The library is already there, we just don't use it yet for some reason:

https://github.com/highlightjs/highlight.js/blob/1936dbd59dcd4de7391a9ea0413a68906fb04c14/package.json#L50

Reply:
I think these (or at least the nodejsone) should be moved to `overrides` and scope to `src/*`, no?  We definitely DO use nodejss modules in tests.  Or the rule could just be removed; I don't see a big risk of Node.js modules accidentally slipping into our client libraries.